repositories/applications_repository.js

```diff
@@ -1,18 +1,19 @@
 const { getKnex } = require("../db");
 
-const knex = getKnex();
-
 async function getAllApplications() {
+  const knex = getKnex();
```

Comment:
I know we discussed moving knex on the file level instead, but in this PR it suddenly stopped working (maybe because of some circular dependency, maybe because of something else)... I propose to keep it like this and dedicate a separate ticket to making it right